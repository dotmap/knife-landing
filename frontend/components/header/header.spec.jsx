/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Header from '.'

describe('Header component', () => {
  it('matches snapshot', () => {
    const component = renderer.create(<Header />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
