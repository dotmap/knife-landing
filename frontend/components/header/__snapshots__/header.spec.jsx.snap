// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<header>
  <h1>
    Next.js Example
  </h1>
</header>
`;
